eth2util/keymanager/keymanager_test.go

```diff
@@ -100,3 +148,25 @@ func TestVerifyConnection(t *testing.T) {
 		require.ErrorContains(t, cl.VerifyConnection(ctx), "parse address")
 	})
 }
+
+// noopKeymanagerReq is a mock keymanager request for use in tests.
+type noopKeymanagerReq struct {
```

Comment:
`mock` not `noop`

Reply:
noop should be just that, no-operation, but you are actualy doing stuff with it. mock is "fake", and can have limited logic and functionality.